cmd/put.go

```diff
@@ -352,13 +352,13 @@ func parsePutFile(path, meta, requester string, splitter bufio.SplitFunc, base64
 }
 
 func parseMetaString(meta string) map[string]string {
-	if meta == "" {
-		return nil
-	}
-
 	kvs := strings.Split(meta, ";")
 	mm := make(map[string]string, len(kvs))
 
+	if meta == "" {
```

Comment:
Missed this before, this if statement is unnecessary as if `meta` is empty, `kvs` will also be empty, meaning that the for statement below will just be skipped and the map returned.

Reply:
Ah, I misread the `Split` description; it only returns an empty slice if the string **and** the seperator and empty strings.